node/cdn/src/blocks.rs

```diff
@@ -444,7 +444,7 @@ mod tests {
 
     type CurrentNetwork = MainnetV0;
 
-    const TEST_BASE_URL: &str = "https://blocks.aleo.org/mainnet/v0";
+    const TEST_BASE_URL: &str = "https://blocks.provable.org/v0/mainnet";
```

Comment:
```suggestion
    const TEST_BASE_URL: &str = "https://cdn.provable.com/v0/blocks/mainnet";
```

This will return a similar error now as the old CDN did so this might work. But also see @raychu86's suggested comment of probably just removing this test.